lib/rubocop/cop/rails/duplicate_association.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for associations that have been defined multiple times in the same file.
+      #
+      # @example
+      #
+      #   # bad
+      #   belongs_to :foo
+      #   belongs_to :bar
+      #   has_one :foo
+      #
+      #   # good
+      #   belongs_to :foo
+      #   has_one :bar
```

Comment:
This example looks `bad` and `good` cases are incompatible. It looks like it should be:

```suggestion
      #   # bad
      #   belongs_to :foo
      #   belongs_to :bar
      #   has_one :foo
      #
      #   # good
      #   belongs_to :bar
      #   has_one :foo
```

How about?

Reply:
🤦🏻‍♂️ Fixed. Thanks